blocks/v2-all-trucks/v2-all-trucks.css

```diff
@@ -117,7 +118,7 @@
 .v2-all-trucks .v2-all-trucks__truck-info > p {
   letter-spacing: .01em;
   margin-top: 12px;
-  text-wrap: balance;
+  text-wrap: pretty;
```

Comment:
`pretty` is supported only by Chromium-based browsers. FF and Safari are not supporting it :(